requirements.txt

```diff
@@ -1,3 +1,5 @@
 ckantoolkit>=0.0.3
-goodtables==1.5.1
+frictionless==5.0.0b9
```

Comment:
Can we use a release version instead of beta?

Reply:
Sure, as soon as frictionless v5 goes out of beta we'll target the stable version